docs/pages/developer-guide.md

```diff
@@ -309,7 +309,7 @@ More powerful custom plans can also be written. Consider the case in which a pod
 
 ```yaml
 name: "hello-world"
-pods:
+pod:
```

Comment:
svc.yml should stay `pods`. unless we're going to be updating `tasks` to `task` too..